plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -52,14 +52,13 @@
   }
 
   @Override
-  public void execute(IProgressMonitor monitor) throws InvocationTargetException, CoreException {
+  public void execute(IProgressMonitor monitor) throws InvocationTargetException, CoreException {    
     SubMonitor progress = SubMonitor.convert(monitor, 100);
     
-    URI location = config.getProject().getLocationURI();
-    
     IWorkspace workspace = ResourcesPlugin.getWorkspace();
     IProject newProject = config.getProject();
-    
+    URI location = newProject.getLocationURI();
```

Comment:
Following on to the previous comment: when 'defaults' is not ticked, then getLocationURI won't be correct.